src/App.js

```diff
@@ -1,29 +1,10 @@
-import logo from './logo.svg';
+import React from 'react';
+import CalculatorInterface from './components/Calculator';
 import './App.css';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
-
-export default App;
+const app = () => (
+  <div className="app">
+    <CalculatorInterface />
+  </div>
+);
+export default app;
```

Comment:
- [ ] It would be better if your App.js component is class-based. Kindly fix this minor issue.
👍